src/routes/examples/main/react.mdx

```diff
@@ -1,3 +1,110 @@
 # React
 
-Coming soon!
+Setting up Mirage with Create React App is easy, it only requires a single configuration file.
```

Comment:
How about

Setting up Mirage with a React app is easy, whether you're using Create React App or some other environment. You only need a single file.